src/components/Form/DateTimeWidget.tsx

```diff
@@ -0,0 +1,49 @@
+import { FormControl, OutlinedInput } from '@mui/material';
+import { WidgetProps } from '@rjsf/utils';
+
+export const DateTimeWidget = ({
+	id,
+	label,
+	value,
+	disabled,
+	name,
+	readonly,
+	autoFocus,
+	placeholder,
+	onBlur,
+	onChange,
+	onFocus,
+	hideLabel,
+	multiple,
+	options,
+	required,
+}: WidgetProps) => {
+	const change = ({ target: { value } }: any) => {
+		return onChange(value === '' ? options.emptyValue : value);
+	};
+	return (
+		<FormControl variant="outlined">
+			<label htmlFor={id}>
+				{label}
+				{required ? '*' : null}
+			</label>
+			<OutlinedInput
+				id={id}
+				type="date"
+				{...{
+					value,
+					disabled,
+					name,
+					readonly,
+					autoFocus,
+					placeholder,
+					hideLabel,
+					multiple,
+				}}
```

Comment:
Why did we specify the full object to destructure instead of just using ...props? For explicitness?

Reply:
we want to specify them as doing {...props} will also add properties that we don't want to pass to the input as props are of type widgetProps which also include schema, uiSchema, registry, rawErrors...